python/setup.py.in

```diff
@@ -31,15 +31,15 @@ pyosdp_sources = '@LIB_OSDP_SOURCES@'.split(';') + [
 
 sources = utils_sources + pyosdp_sources
 
-libosdp_include_dirs = ('@LIB_OSDP_INCLUDE_DIRS@'.split(';') +
+libosdp_include_dirs = ('@PROJECT_SOURCE_DIR@/utils/include'.split(';') +
+                        '@PROJECT_SOURCE_DIR@/include'.split(';') +
```

Comment:
Why not just set LIB_OSDP_INCLUDE_DIRS below and leave this as it is now? If there is a reason for this change, then LIB_OSDP_INCLUDE_DIRS is no longer used anywhere, so we must get rid of it.

Reply:
I had removed `LIB_OSDP_INCLUDE_DIRS` because of problems with how it was defined, directories were included more than once and CMake and/or vcpkg was not happy about it.